src/renderer/components/ft-icon-button/ft-icon-button.vue

```diff
@@ -33,12 +37,19 @@
         <ul
           v-if="dropdownOptions.length > 0"
           class="list"
+          role="listbox"
+          aria-expanded="false"
```

Comment:
The expanded property should be updated when the drop down is shown and hidden.
```suggestion
          :aria-expanded="dropdownShown"
```